.github/scripts/authors_in_cff.py

```diff
@@ -2,11 +2,11 @@
 
 import os
 import pathlib
-import requests
 import sys
-
 from typing import Optional
```

Comment:
With the default settings:
 - Standard library imports go first (which I really like!)
 - In a particular section, `from` imports follow `import` imports with no line between them

Reply:
😄